x-pack/solutions/observability/plugins/profiling/common/setup_api_schema.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const profilingSetupStatusResponseSchema = schema.object({
```

Comment:
I don't see this being used anywhere in the code, if we can remove this, it'll solve the bundle size problem.